job-server-tests/src/main/java/spark.jobserver/JavaHelloWorldJob.java

```diff
@@ -1,12 +1,12 @@
-package spark.jobserver;
+// package spark.jobserver;
```

Comment:
Hmm, I think you may be on the right track.  

@hntd187 would you have time to try pushing your idea of fix to this branch?  

> On Jun 1, 2016, at 1:35 PM, Stephen Carman <notifications@github.com> wrote:
> 
> In job-server-tests/src/main/java/spark.jobserver/JavaHelloWorldJob.java <https://github.com/spark-jobserver/spark-jobserver/pull/442#discussion_r65437194>:
> 
> > @@ -1,12 +1,12 @@
> > -package spark.jobserver;
> > +// package spark.jobserver;
> @velvia <https://github.com/velvia> @zeitos <https://github.com/zeitos> It should be something like this, but the jobserver itself needs to account for the different class. The type C stuff as well as the case object do not translate well to java, so they don't compile. But also take note of the path I changed as also Zeitos suggested...
> 
> hntd187@825d02e <https://github.com/hntd187/spark-jobserver/commit/825d02e7051dd73ed5c2acf77a6836e6efb9950a>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/spark-jobserver/spark-jobserver/pull/442/files/6c9e4d963c90a8f9ce50b818440d3f2c16149699#r65437194>, or mute the thread <https://github.com/notifications/unsubscribe/ABA323oifnIGeAJQ29XdekUSzbasVdw7ks5qHe0XgaJpZM4IQq8l>.
>